data/baruKreddy.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Bhargavi Reddy K",
+  "type": "personal",
+  "bio": "**OpenSource Developer**. Travel enthusiast & People Lover"
```

Comment:
Hi @baruKreddy, please add a **'Comma'** at the end of this line. :+1:

Reply:
Sure thing, Thank you :)